vuepress/docs/next/tutorials/solution/install-standard-demo.md

```diff
@@ -62,120 +89,142 @@ ent ecr deploy --repo="https://github.com/entando-samples/standard-demo-manage-u
 
-In the unlikely event you encounter conflicts during an initial installation, you will be presented with an Installation Plan like the one shown below. Select `Update All` in the upper right after making your selections.
-   ![InstallPlan.png](./images/InstallPlan.png)
+     **Order of installation is important.** The `standard-demo-content-bundle` must be installed last, as it relies on MFEs from the other bundles to set up each of the pages.
 
+     >Note: The command line deploys and installs the Standard Banking Demo bundles without thumbnails.
 
-5. Access the Standard Banking Demo via one of the following options:
+     >Note: Each installation can take several minutes while the application downloads Linux images for the microservices and installs related assets. The `standard-demo-banking-bundle` and `standard-demo-customer-bundle` include microservices that require container initializations and take longer to install.
 
-**Option 1** If you'd like to make the Standard Banking Demo your default home page, go to `App Builder → Pages → Settings`. In
-   the dropdown for Home Page, select `Home / Home SD` and click `Save`.
-   ![HomepageSelect.png](./images/HomepageSelect.png)
+     ::: tip
+     In the unlikely event of conflicts during an initial installation, you will be presented with an Installation Plan like the one shown below. Select `Update All` in the upper right after making your selections.
+     ![InstallPlan.png](./images/InstallPlan.png)
+     :::
 
-You can now navigate to your application's home page using the home icon in the upper right of the App Builder.
+## Access the Standard Banking Demo
 
+Access the Standard Banking Demo via one of the following options.
 
-**Option 2** Alternatively, you can view the Standard Banking Demo home page by going to `Pages → Management`, finding `Home SD` in the page tree, and clicking `View Published Page` from its actions.
+### Option 1
 
+Make the Standard Banking Demo your default home page. The home icon shortcut will then navigate to the Standard Banking Demo home page. 
+
+1. Go to `App Builder` → `Pages` → `Settings` 
+2. In the dropdown for Home Page, select `Home / Home SD`
+3. Click `Save`
+![HomepageSelect.png](./images/HomepageSelect.png)
+4. Click the home icon in the upper right of the App Builder
+
+
+### Option 2 
+
+Navigate to the Standard Banking Demo home page from your App Builder page tree.
+
+1. Go to `App Builder` → `Pages` → `Management` 
+2. Find `Home SD` in the page tree
+3. From the `Actions` pull-down menu, go to `View Published Page`
 ![Standard Banking Demo Homepage](./images/Homepage.png)
 
 
 ## Application Details
 
-The Entando Standard Banking Demo application demonstrates a number of the major features of the Entando platform, including:
+The Entando Standard Banking Demo application demonstrates a number of the major features of the Entando Platform:
 
 * Keycloak integration for role based access controls
```

Comment:
role-based

Reply:
we decided on no hyphens when doing the rbac page so staying consistent